src/components/data/DataTable.js

```diff
@@ -243,9 +245,8 @@ const DataTable = props => {
                       setColumnWidths(_.set('name', nameWidth + delta))
                     }
                   }, [
-                    h(Sortable, { sort, field: 'name', onSort: setSort }, [
-                      h(HeaderCell, [entityMetadata[entityType].idName])
-                    ])
+                    h(HeaderOptions, { field: 'name', onSort: setSort, isEntityName: true },
+                      [h(HeaderCell, [entityMetadata[entityType].idName])])
```

Comment:
Two comments on this menu:
1. We've lost the up/down arrow that indicates which column is currently the active sort order - that icon was in `Sortable` but is not in `HeaderOptions`. I think we should have that back.


| old | new |
| --- | --- |
| ![sort-old](https://user-images.githubusercontent.com/6041577/137141805-b1a67e2b-143c-44f2-b784-f05ce66744c9.png) | ![sort-new](https://user-images.githubusercontent.com/6041577/137141820-83083f62-8a81-43ad-ab38-7c6a61087fb5.png)
 |




2. At least in the case where a column is not the active sort order, we've lost some characters of the column's header. Is it possible to move the menu a bit to the right to gain back any characters we can? Those are precious and valuable:


| old | new |
| --- | --- |
| ![colname-old](https://user-images.githubusercontent.com/6041577/137141578-513bbc41-7f1b-4d79-96bc-d756d05a0105.png) | ![colname-new](https://user-images.githubusercontent.com/6041577/137141590-3e441bee-ae13-46c3-8dcc-3de2897bc1b2.png) |